### ISSUE TITLE
Option for the menu to be in 240p

### ISSUE THREAD

**Issue by v1605** [Polprzewodnikowy/N64FlashcartMenu#82]
### Description

To my eye using the the N64Digital scaling to 1080p, the 240p everdrive menu is a lot more clear than the 480i menu present here.

### How to solve the problem

If there was an option to set the resolution to 240p in the config that would really help and allow others to keep the 480i if they want.

**Comment by TharathielCB** [comment]
This could also close issue https://github.com/Polprzewodnikowy/N64FlashcartMenu/issues/59 where the deinterlacer destabilise the picture.

**Comment by v1605** [comment]
Not quite the same, I'm using motion adaptive deinterlacing on a NTSC system (clock mod also enabled on the n64digital) so the stabilization is not an issue, it more that 240p seems to scale better at least to my eye.

**Comment by TharathielCB** [comment]
Yes, but I'm saying that changing to 240p would solve issue #59. Not that it is the same issue.

**Comment by v1605** [comment]
Got it, yes that would solve it.

**Comment by Polprzewodnikowy** [comment]
While I appreciate more options myself, redoing whole menu to operate at 240p or giving user an option to choose between 240p/480i is not something that could be done easily with current components and the way element positioning is implemented now.

If you're using any upscaler or HDMI mod I strongly recommend setting deinterlacing on those devices to `weave` or `motion adaptive` option. `Bob` deinterlacing will unfortunately cause screen to vibrate vertically with rather unpleasant look.

I'm going to leave this issue open, it would be nice to have an option but it's not a priority now.

**Comment by riggles1** [comment]
A lot of us CRT users really don't like 480i either, it's very flickery looking at the edges especially on big sharp CRT's and the N64 not applying any vertical filtering to try hide this. Glad it's not dismissed completely, even if the 240p option becomes a basic looking menu, that'd still be great and preferable. 
Looking forward to this issue being revisited some day.

**Comment by spspspspspsp** [comment]
+1. 640x480i menus are really hard on the eyes when viewed on a CRT. The 240p option on the old ED64 was great for that reason.

**Comment by SubElement** [comment]
+1, just got my SC64 recently and would love a 240p menu also.

**Comment by networkfusion** [comment]
The ED64 X7 and Pro uses a resolution of 640x240. That might be an easier option.
